cgsmiles/tests/test_cgsmile_parsing.py

```diff
@@ -34,6 +34,36 @@
                         ["PMA", "PEO", "PMA"],
                         [(0, 1), (1, 2), (0, 2)],
                         [1, 1, 1]),
+                        # smiple cycle sequence bond order to next
+                        ("{[#PMA]1=[#PEO][#PMA]1}",
+                        ["PMA", "PEO", "PMA"],
+                        [(0, 1), (1, 2), (0, 2)],
+                        [2, 1, 1]),
+                        # smiple cycle sequence bond order in cycle
+                        ("{[#PMA]=1[#PEO][#PMA]1}",
+                        ["PMA", "PEO", "PMA"],
+                        [(0, 1), (1, 2), (0, 2)],
+                        [1, 1, 2]),
+                        # smiple cycle sequence two bond orders
+                        ("{[#PMA].1=[#PEO][#PMA]1}",
+                        ["PMA", "PEO", "PMA"],
+                        [(0, 1), (1, 2), (0, 2)],
+                        [2, 1, 0]),
+                        # smiple cycle sequence with % bond order
+                        ("{[#PMA]=%123[#PEO][#PMA]%123}",
```

Comment:
I think smiles ring bonds only go up to '%99' (and note that '%01' and '1' refer to the same bond)

Reply:
huh; do we need that limitation? I'll take care of the 01 vs 1

Reply:
It's an important distinction to make: `%123` could mean ring index 123, or ring indices 12 and 3. I'm fine with either, but with a small preference for sticking to existing smiles conventions